src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -93,6 +100,277 @@ function TimePeriodSelector(props) {
+      });
+    },
+    [countryId, behavioralResponseReforms],
+  );
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [isChecked, setIsChecked] = useState(hasBehavioralResponses(policy));
+  const spellsBehaviour = ["uk", "ca"].includes(countryId);
+  const displayCategory = useDisplayCategory();
+
+  // Ref for storing previous label in case user reverts
+  const prevLabelRef = useRef(null);
+
+  useEffect(() => {
+    setIsChecked(hasBehavioralResponses(policy));
+  }, [policy, hasBehavioralResponses]);
+
+  async function handleChange(value) {
+    if (value) {
+      await setBehavioralResponses();
+    } else {
+      await removeBehavioralResponses();
+    }
+
+    setIsChecked(value);
+  }
+
+  async function setBehavioralResponses() {
+    // Save the previous reform's label in case user reverts
+    prevLabelRef.current = policy.reform.label;
+    const policyToStack = behavioralResponseReforms[metadata.countryId];
+
+    // Fetch policy to stack
+    let newPolicyData = {
+      ...policy.reform.data,
+      ...policyToStack,
+    };
+
+    // Create new policy with those parameters and emit to back end, receiving back ID
+    const newPolicyRes = await getNewPolicyId(
+      metadata.countryId,
+      newPolicyData,
+    );
+    let newSearch = copySearchParams(searchParams);
+    newSearch.set("reform", newPolicyRes.policy_id);
+    setSearchParams(newSearch);
+  }
+
+  async function removeBehavioralResponses() {
+    const policyToStack = behavioralResponseReforms[metadata.countryId];
+
+    // Fetch policy to stack
+    let newPolicyData = {
+      ...policy.reform.data,
+    };
+
+    for (const key of Object.keys(policyToStack)) {
+      delete newPolicyData[key];
+    }
+
+    let newPolicyLabel = null;
+    // If there's a previous label, revert to it
+    if (prevLabelRef.current) {
+      newPolicyLabel = prevLabelRef.current;
+    }
+
+    // Current law is represented by an empty object;
+    // if the new policy is empty, remove the reform query parameter
+    if (Object.keys(newPolicyData).length === 0) {
+      let newSearch = copySearchParams(searchParams);
+      newSearch.delete("reform");
+      setSearchParams(newSearch);
+      return;
+    }
+
+    // Create new policy with those parameters and emit to back end, receiving back ID
+    const newPolicyRes = await getNewPolicyId(
+      metadata.countryId,
+      newPolicyData,
+      newPolicyLabel,
+    );
+    let newSearch = copySearchParams(searchParams);
+    newSearch.set("reform", newPolicyRes.policy_id);
+    setSearchParams(newSearch);
+  }
+
+  // Don't show behavior response toggle for countries without
+  // policies to set them
+  if (!Object.keys(behavioralResponseReforms).includes(countryId)) {
+    return null;
+  }
+
+  const preLabel = {
+    us: "CBO",
+  };
+
+  // Apply the correct label based on the country
+  const tooltipDesc = {
+    uk:
+      "When selected, simulations adjust earnings and capital gains using " +
```

Comment:
We're not using CBO for capital gains so how about just
>When selection, simulations adjust earnings and capital gains based on how reforms affect net income and marginal tax rates.

and for the US:
>>When selection, simulations adjust earnings and capital gains based on how reforms affect net income and marginal tax rates, applying the Congressional Budget Office's assumptions.

Maybe with "Learn more" at the end if it's not apparent from the tooltip

Reply:
This is great. I also no longer have to choose between "behavioral" and "behavioural".